src/ngraph/opsets/opset1_tbl.hpp

```diff
@@ -53,6 +53,7 @@
 NGRAPH_OP(Abs, ngraph::op::v0)
 NGRAPH_OP(Acos, ngraph::op::v0)
 NGRAPH_OP(Add, ngraph::op::v1)
+NGRAPH_OP(ArgMax, ngraph::op::v0) // ArgMax is not in opset1 spec
```

Comment:
If `ArgMax` is not in the opset1 spec it should be added to it.
We can't add random ops to opsets.

Reply:
I just wanted to show that, this is why I left this comment here. Removed.